### DISCUSSION TITLE
Listing Seasonal Anime

### DISCUSSION THREAD

**Discussion by Korkut-yildiz** [pystardust/ani-cli#1290] [Ideas] [answered]
I think It'd be nice see the anime's that is airing in the season and which day' and hour it is going to be released.

**Comment by 71zenith** [comment, marked as answer]
ani-cli is a minimalist tool . it will not be adding such functionality. u might want to resort to some anime rss tool such as https://github.com/CoolnsX/anime-rss for arrival updates
